processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByQueryEngineV2.java

```diff
@@ -153,10 +175,10 @@ public void close() throws IOException
     private final ByteBuffer buffer;
     private final Grouper.KeySerde<ByteBuffer> keySerde;
     private final DateTime timestamp;
-    private final DimensionSelector[] selectors;
     private final ByteBuffer keyBuffer;
     private final int[] stack;
-    private final IndexedInts[] valuess;
+    private final Object[] valuess;
```

Comment:
it's an array of row values, so I guess it's a fun plural for "values"? You'll have to ask @gianm for the naming reason :)